user/common-tasks/usb-devices.md

```diff
@@ -12,6 +12,8 @@ redirect_from:
 
 If you are looking to handle USB *storage* devices (thumbdrives or USB-drives), please have a look at the [block device] page.
 
+**Note:** you cannot pass through devices from dom0 (in other words: a [USB qube][USB-qube howto] is required).
```

Comment:
I think this sentence is very confusing when removed from its previous context (and perhaps it was already confusing before). Seeing it near the top of the page like this, I'm not even sure how to interpret it.

Reply:
If I were a user reading this page for the first time, after reading this sentence, I would probably wonder:
 - Pass through to where?
 - Why can't I pass through devices from dom0 if that's where they are? How else am I supposed to get them out?
 - A USB qube is required for what?

Reply:
Perhaps it is less confusing to rephrase the sentence to something like the following?

> Attaching USB devices to VMs requires a [USB qube](USB-qube howto).

One can also move the sentence to the "Creating And Using a USB qube" section, though again the visibility may not be as high.

(For the record, my original justification for placing the sentence there untouched was that, as the term "USB passthrough" was used quite frequently in the first section and throughout the page, I ended up assuming the concept to be prior knowledge. I understand the concerns though.)

Reply:
I think your new proposed sentence is much clearer and simpler.

Reply:
The updated version is pushed.